static/app/views/issueList/issueListTable.tsx

```diff
@@ -133,7 +114,7 @@ function IssueListTable({
                 displayReprocessingLayout={displayReprocessingActions}
                 query={query}
                 selectedProjectIds={selection.projects}
-                loading={issuesLoading}
+                loading={issuesLoading || (statsLoading && !groupIds.length)}
```

Comment:
nit: would be nice if you could leave a comment that explains why we need to check `statsLoading && !groupIds.length` (because `groupIds` doesn't update immediately)